libs/sdk-core/src/binding.rs

```diff
@@ -304,28 +304,28 @@ pub fn parse(s: String) -> Result<InputType> {
     block_on(async { crate::input_parser::parse(&s).await })
 }
 
-/// See [BreezServices::pay_lnurl]
+/// See [BreezServices::lnurl_pay]
 pub fn pay_lnurl(
     user_amount_sat: u64,
     comment: Option<String>,
     req_data: LnUrlPayRequestData,
 ) -> Result<LnUrlPayResult> {
     block_on(async {
         get_breez_services()?
-            .pay_lnurl(user_amount_sat, comment, req_data)
+            .lnurl_pay(user_amount_sat, comment, req_data)
             .await
     })
 }
 
-/// See [BreezServices::withdraw_lnurl]
+/// See [BreezServices::lnurl_withdraw]
```

Comment:
I think we should rename to lnurl_withdraw in the bindings.rs as well

Reply:
Good catch. Done.